Capri.Web/Controllers/CourseController.cs

```diff
@@ -54,6 +54,11 @@ public IActionResult GetAll()
         public async Task<IActionResult> Create(
             [FromBody] CourseRegistration registration)
         {
+            if(registration == null || !ModelState.IsValid)
+            {
+                return BadRequest("You provided invalid data");
```

Comment:
Same as here, impersonal form will be better.

Reply:
Done